rskj-core/src/test/java/org/ethereum/net/client/CapabilityTest.java

```diff
@@ -0,0 +1,41 @@
+package org.ethereum.net.client;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class CapabilityTest {
+
+    @Test
+    public void testEquals_OK() {
```

Comment:
I'd also add similar tests for the case of the `ProgramInvokeImpl` class

Reply:
Done!